flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/serde/ContextResolvedTableSerdeTest.java

```diff
@@ -191,6 +176,24 @@ void withAnonymousTable() {
                             TableConfigOptions.CatalogPlanCompilation.IDENTIFIER,
                             TableConfigOptions.CatalogPlanRestore.IDENTIFIER);
 
+            @Test
+            void withAnonymousTable() throws Exception {
```

Comment:
how about we move these tests for anonymous tables out of the nested test structure to top-level test methods? anonymous tables have nothing to do with the test matrix for catalog objects.